<robot name="exp_table">
  <link name="table">
    <inertial>
      <origin xyz="0.61 1.221 0.0095"/>
      <mass value="100.0" />
      <inertia  ixx="0.0" ixy="0.0"  ixz="0.0"  iyy="0.0"  iyz="0.0"  izz="0.0" />
    </inertial>
    <visual>
      <origin xyz="0.61 1.221 0.0095"/>
      <geometry>
        <box size="1.22 2.442 0.019" />
      </geometry>
    </visual>
    <collision>
      <origin xyz="0.61 1.221 0.0095"/>
      <geometry>
        <box size="1.22 2.442 0.019" />
      </geometry>
    </collision>
  </link>

  <link name="leg_1">
    <inertial>
      <origin xyz="0.03 0.03 0.4325"/>
      <mass value="10.0" />
      <inertia  ixx="0.0" ixy="0.0"  ixz="0.0"  iyy="0.0"  iyz="0.0"  izz="0.0" />
    </inertial>
    <visual>
      <origin xyz="0.03 0.03 0.4325"/>
      <geometry>
        <cylinder length="0.865" radius="0.03"/>
      </geometry>
    </visual>
    <collision>
      <origin xyz="0.03 0.03 0.4325"/>
      <geometry>
        <cylinder length="0.865" radius="0.03"/>
      </geometry>
    </collision>
  </link>

  <link name="leg_2">
    <inertial>
      <origin xyz="0.03 0.03 0.4325"/>
      <mass value="10.0" />
      <inertia  ixx="0.0" ixy="0.0"  ixz="0.0"  iyy="0.0"  iyz="0.0"  izz="0.0" />
    </inertial>
    <visual>
      <origin xyz="0.03 0.03 0.4325"/>
      <geometry>
        <cylinder length="0.865" radius="0.03"/>
      </geometry>
    </visual>
    <collision>
      <origin xyz="0.03 0.03 0.4325"/>
      <geometry>
        <cylinder length="0.865" radius="0.03"/>
      </geometry>
    </collision>
  </link>

  <link name="leg_3">
    <inertial>
      <origin xyz="0.03 0.03 0.4325"/>
      <mass value="10.0" />
      <inertia  ixx="0.0" ixy="0.0"  ixz="0.0"  iyy="0.0"  iyz="0.0"  izz="0.0" />
    </inertial>
    <visual>
      <origin xyz="0.03 0.03 0.4325"/>
      <geometry>
        <cylinder length="0.865" radius="0.03"/>
      </geometry>
    </visual>
    <collision>
      <origin xyz="0.03 0.03 0.4325"/>
      <geometry>
        <cylinder length="0.865" radius="0.03"/>
      </geometry>
    </collision>
  </link>

  <link name="leg_4">
    <inertial>
      <origin xyz="0.03 0.03 0.4325"/>
      <mass value="10.0" />
      <inertia  ixx="0.0" ixy="0.0"  ixz="0.0"  iyy="0.0"  iyz="0.0"  izz="0.0" />
    </inertial>
    <visual>
      <origin xyz="0.03 0.03 0.4325"/>
      <geometry>
        <cylinder length="0.865" radius="0.03"/>
      </geometry>
    </visual>
    <collision>
      <origin xyz="0.03 0.03 0.4325"/>
      <geometry>
        <cylinder length="0.865" radius="0.03"/>
      </geometry>
    </collision>
  </link>

  <link name="leg_5">
    <inertial>
      <origin xyz="0.03 0.03 0.4325"/>
      <mass value="10.0" />
      <inertia  ixx="0.0" ixy="0.0"  ixz="0.0"  iyy="0.0"  iyz="0.0"  izz="0.0" />
    </inertial>
    <visual>
      <origin xyz="0.03 0.03 0.4325"/>
      <geometry>
        <cylinder length="0.865" radius="0.03"/>
      </geometry>
    </visual>
    <collision>
      <origin xyz="0.03 0.03 0.4325"/>
      <geometry>
        <cylinder length="0.865" radius="0.03"/>
      </geometry>
    </collision>
  </link>

  <link name="leg_6">
    <inertial>
      <origin xyz="0.03 0.03 0.4325"/>
      <mass value="10.0" />
      <inertia  ixx="0.0" ixy="0.0"  ixz="0.0"  iyy="0.0"  iyz="0.0"  izz="0.0" />
    </inertial>
    <visual>
      <origin xyz="0.03 0.03 0.4325"/>
      <geometry>
        <cylinder length="0.865" radius="0.03"/>
      </geometry>
    </visual>
    <collision>
      <origin xyz="0.03 0.03 0.4325"/>
      <geometry>
        <cylinder length="0.865" radius="0.03"/>
      </geometry>
    </collision>
  </link>

 <link name="world"/>
 <joint name="table_fixed" type="fixed">
    <parent link="world"/>
	<child link="table"/>
 </joint>

 <joint name="table_to_leg_1" type="fixed">
    <parent link="table"/>
    <child link="leg_1"/>
    <origin xyz="0.14 0.19 -0.865"/>
  </joint>

 <joint name="table_to_leg_2" type="fixed">
    <parent link="table"/>
    <child link="leg_2"/>
    <origin xyz="0.14 1.191 -0.865"/>
  </joint>

 <joint name="table_to_leg_3" type="fixed">
    <parent link="table"/>
    <child link="leg_3"/>
    <origin xyz="0.14 2.201 -0.865"/>
  </joint>

  <joint name="table_to_leg_4" type="fixed">
    <parent link="table"/>
    <child link="leg_4"/>
    <origin xyz="1.02 0.19 -0.865"/>
  </joint>
 
  <joint name="table_to_leg_5" type="fixed">
    <parent link="table"/>
    <child link="leg_5"/>
    <origin xyz="1.02 1.191 -0.865"/>
  </joint>
 
  <joint name="table_to_leg_6" type="fixed">
    <parent link="table"/>
    <child link="leg_6"/>
    <origin xyz="1.02 2.201 -0.865"/>
  </joint>
 
  <gazebo reference="table">
    <material>Gazebo/White</material>
     <mu1>1000</mu1>
     <mu2>1000</mu2>
  </gazebo>
</robot>
